dev.geopjr.Turntable.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: dev.geopjr.Turntable
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: dev.geopjr.Turntable
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.mpris.MediaPlayer2.*
+  - --talk-name=org.freedesktop.DBus
+  - --filesystem=~/.mozilla/firefox/firefox-mpris/:ro
+  - --filesystem=~/.var/app/org.mozilla.firefox/.mozilla/firefox/firefox-mpris/:ro
+  - --filesystem=~/.librewolf/firefox/firefox-mpris/:ro
+  - --filesystem=~/.var/app/io.gitlab.librewolf-community/data/firefox-mpris/:ro
+  - --filesystem=~/.floorp/firefox/firefox-mpris/:ro
+  - --filesystem=~/.var/app/app.zen_browser.zen/data/firefox-mpris/:ro
+  - --filesystem=~/.zen/firefox/firefox-mpris/:ro
+  - --filesystem=~/.var/app/one.ablaze.floorp/data/firefox-mpris/:ro
+  - --filesystem=~/.var/app/io.bassi.Amberol/cache/amberol/covers/:ro
+  - --filesystem=~/.var/app/com.github.neithern.g4music/cache/com.github.neithern.g4music/:ro
+  - --filesystem=~/.var/app/ca.edestcroix.Recordbox/data/Recordbox/:ro
+  - --filesystem=~/.var/app/org.gnome.Rhythmbox3/cache/rhythmbox/album-art/:ro
+  - --filesystem=~/.var/app/org.gnome.Lollypop/cache/lollypop/:ro
+  - --filesystem=~/.cache/:ro
```

Comment:
what is whole `~/.cache` for?

Reply:
Apps installed as packages (like GNOME Music in many distros), use .cache

Reply:
Though I don't know if all these filesystems permissions are worth it, if in the end I'll have to get `host` for the MPRIS app icons